languages/java/exercises/concept/arrays/src/test/java/BirdCountTest.java

```diff
@@ -0,0 +1,77 @@
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.Ignore;
+
+
+import static org.junit.Assert.assertEquals;
+
+public class BirdCountTest {
+
+    private BirdCount birdCount;
+    private int lastWeek[] = {0, 2, 5, 3, 7, 8, 4};
+
+    @Before
+    public void setUp() {
+        birdCount = new BirdCount(lastWeek);
+    }
+
+    @Test
+    public void itTestGetLastWeek() {
+        assertEquals(true, birdCount.getLastWeek().equals(lastWeek));
+    }
+
+    @Test
```

Comment:
The `@Ignore` annotation should be applied to all but the first test case. This comment also applies to the other (non-first) test cases.